specification/ai/Azure.AI.Unified/datasets/routes.tsp

```diff
@@ -0,0 +1,136 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Foundations;
+
+namespace Azure.AI.Unified;
+
+interface Datasets {
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("List latest version of each dataset in a project.")
+  @route("/datasets")
+  @get
+  listLatestDatasets(
+    @doc("Continuation token for pagination. This is the nextLink from the previous response.")
+    @query("$continuationToken")
+    $continuationToken?: string,
```

Comment:
You have both a nextLink and a continuation token. One should be enough.

Reply:
The python signature is this at the moment:
```python
   def list_latest_datasets(
        self,
        *,
        continuation_token_parameter: Optional[str] = None,
        list_view_type: Optional[Union[str, _models.ListViewType]] = None,
        **kwargs: Any
    ) -> Iterable["_models.DatasetVersion"]:
```